lightning/src/onion_message/messenger.rs

```diff
@@ -260,21 +260,30 @@ pub struct Responder {
 
 impl Responder {
 	/// Creates a new [`Responder`] instance with the provided reply path.
-	fn new(reply_path: BlindedPath, path_id: Option<[u8; 32]>) -> Self {
+	pub(super) fn new(reply_path: BlindedPath, path_id: Option<[u8; 32]>) -> Self {
 		Responder {
 			reply_path,
 			path_id,
 		}
 	}
 
-	/// Creates the appropriate [`ResponseInstruction`] for a given response.
+	/// Creates a [`ResponseInstruction::WithoutReplyPath`] for a given response.
```

Comment:
How about we add a second sentence? Just want to avoid repeating the linked docs.

```rust
/// Use when the recipient doesn't need to send back a reply.
```

Similarly for the other method.

Reply:
I think this line will aptly explain the purpose of this function without introducing a risk of future docs desync.
Thanks for the suggestion!
Updated docs in [57bca64](https://github.com/lightningdevkit/rust-lightning/commit/57bca64bc1b7b3bbfc34fffa395cf9a38d5f8d48)